core/admin/mailu/manage.py

```diff
@@ -96,7 +97,7 @@ def admin(localpart, domain_name, password, mode):
 def user(localpart, domain_name, password):
     """ Create a user
     """
-    domain = models.Domain.query.get(domain_name)
+    domain = db.session.get(models.domain, domain_name)
```

Comment:
This fails during the automatic test with the error:
```
   File "/app/mailu/manage.py", line 100, in user
    domain = db.session.get(models.domain, domain_name)
                            ^^^^^^^^^^^^^
AttributeError: module 'mailu.models' has no attribute 'domain'. Did you mean: 'Domain'?
```

full stack trace at: https://github.com/Mailu/Mailu/actions/runs/6868358963/job/18678819073#step:11:1197

Reply:
This should be `models.Domain` indeed, fixed.